src/components/tools/Stats/AvgRange.svelte

```diff
@@ -24,18 +25,25 @@
   export let periodList;
 
   /** Id of default group to initialize StatPanel. This prop is optional. Defaults to first item in groupList */
-  export let groupId;
+  export let groupId = groupList[0].id;
```

Comment:
this will throw an error if `groupList` is `undefined`

Reply:
fixed in https://github.com/berkeley-gif/caladapt-website-2021/pull/128/commits/4add49f56dc12f2c1cefb201e7f6168a8081ac54